packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -101,29 +100,28 @@ export function autocomplete<TItem extends BaseItem>(
 
   function runRender() {
     const renderProps = {
-      isTouch: isTouch.value,
-      state: lastStateRef.current,
       autocomplete: autocomplete.value,
-      propGetters,
-      dom: dom.value,
+      autocompleteScopeApi,
       classNames: props.value.renderer.classNames,
+      container: props.value.renderer.container,
+      createElement: props.value.renderer.renderer.createElement,
```

Comment:
according to the docs, it's `render.renderer`, but here it's `renderer.renderer`, do I misread something?

Reply:
The docs state that it's [`renderer`](https://deploy-preview-381--algolia-autocomplete.netlify.app/docs/autocomplete-js/#renderer). The line you're commenting is an internal that defaults and separates the options from core and renderer (the name was here before calling this new option `renderer`).

I'll refactor the way to default options in the future anyway, which will removes this ambiguity.